### PR TITLE
Utility for converting `v14` to `v13` 

### PR DESCRIPTION
**Pull request by ascjones**
In order to ease the transition for any clients in the ecosystem which depend on the existing `v13` metadata format, provide a utility to convert between `v14` and `v13` metadata.

### CODE REVIEW

File: utils/Cargo.toml [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+
+[features]
+default = [
+    "frame-metadata/v14",
+    "frame-metadata/v13",
+    "std",
+]
+std = [
+    "frame-metadata/std",
+]
```
Comment (chevdor) [inline utils/Cargo.toml:24]: eof :-p

**Comment by bkchr** [comment]
How should this work? If someone needs to update the crate, they already get v14? Why should they downgrade that to v13 then?

**Comment by ascjones** [comment]
It's for any downstream consumers who depend on `v13` metadata, at least to allow them to continue to work while they update to `v14`. 

I'm not aware of the whole ecosystem of metadata consumers, I'm just wary that they have not spotted my PR and suddenly their stuff will break. It may not be needed to ease the transition, but at least it is available.

Also allows comparisons between runtimes with `v13` and `v14` metadata, just run the conversion and compare `v13` against `v13`, which could be useful for diffing tooling that @chevdor is working on.

**Comment by bkchr** [comment]
How should downstream projects use this? I would argue that most downstream projects are using js anyway :P 

Maybe we should find out who are our downstream users.

**Comment by ascjones** [comment]
> How should downstream projects use this?

I'll provide a CLI tool which takes the encoded `v14` bytes/json and returns the `v13` bytes/json. If I make it `no-std` it could even be incorporated into `api-sidecar`, but I don't think we want to support that.

> I would argue that most downstream projects are using js anyway :P

That is my impression too, and I really hope so. Let's call this an insurance policy. Happy to throw this away if it is not needed,

In the end I can run it as a one off verification for my PR by comparing `master` (v13) with metadata produced by my PR branch.

**Comment by bkchr** [comment]
> I'll provide a CLI tool which takes the encoded `v14` bytes/json and returns the `v13` bytes/json. If I make it `no-std` it could even be incorporated into `api-sidecar`, but I don't think we want to support that.

Ahh nice. Okay. I don't have seen this. Fine :P I shut up :D

**Comment by ascjones** [comment]
Ecosystem has already transitioned to `V14` and been on it for a while so this is probably not required now.

**Pull request closed without merging**